### ISSUE TITLE
Publish on Serverless Application Repository

### ISSUE THREAD

**Issue by ChristopheBougere** [PokaInc/cloudformation-ami#1]
Hi,

Thanks for sharing this, it took me a few days to understand that the other solutions I was trying weren't actually working when updating the stack.
I am currently trying with this stack and it seems way easier!

Have you thought about publishing it on [Publish on Serverless](https://aws.amazon.com/fr/serverless/serverlessrepo/)?
That way we could deploy it in one click.
Definitely not a big issue but it could be great not having to clone or vendor the repo, and I think this project would totally fit to the [Serverless Application Model](https://github.com/awslabs/serverless-application-model).

**Comment by spg** [comment]
I haven't looked into the Serverless Application Repository, but like you say, it might fit in there!
I'll investigate further, thanks for the suggestion.

**Comment by zaro0508** [comment]
I agree, publishing to Serverless App Repo would be great but another option is to just use github as your repo.  You can do that by downloading the scripts first then using the console/awscli to deploying them.  We use [scepter's hooks](https://sceptre.cloudreach.com/latest/docs/hooks.html) instead of the awscli to share templates.   Probably the only request is to have @spg create github releases so we can get versioned templates.
